### PR TITLE
Improving the list walls tool

### PR DESCRIPTION
**Pull request by jmcouffin**
- Changed the lvl_param = view.LookupParameter("Associated Level") to make it language proof
- added a basic configurator to set offsets, textnote type, and description display
- added option to add the description (widht, layer, material)

### CODE REVIEW

File: pyChilizer.tab/Template.panel/template2.stack/List Walls.pushbutton/script.py [hunk 1]
```diff
@@ -108,19 +139,16 @@ def change_wall_type(wall, wall_type):
 view = revit.active_view
 level = get_level_from_view(view)
 
-line_length = 6
```
Comment (jmcouffin) [inline pyChilizer.tab/Template.panel/template2.stack/List Walls.pushbutton/script.py:111]: moved this to the top as CONSTANTS grabbed from the config file

Comment (dnenov) [reply pyChilizer.tab/Template.panel/template2.stack/List Walls.pushbutton/script.py:111]: Hey man! Thanks for the PR! 

Let me review and will merge.

Comment (dnenov) [reply pyChilizer.tab/Template.panel/template2.stack/List Walls.pushbutton/script.py:111]: Options do come up, but appear blank @jmcouffin . I have just tested it with the Simple Revit Sample project. Maybe something I am not getting in terms of how it is to be used?

image

**Comment by dariaivanciucova** [comment]
hey @jmcouffin apologies, I know this has been too long on the waitlist. 
I just wanted to double check what's the purpose of this tool and how you usually use it. 
I'm confused because I remember it to be creating legend components instead of actual walls in the model. Which option would actually be more useful in your opinion - legend component walls or model walls?

**Comment by jmcouffin** [comment]
> hey @jmcouffin apologies, I know this has been too long on the waitlist. I just wanted to double check what's the purpose of this tool and how you usually use it. I'm confused because I remember it to be creating legend components instead of actual walls in the model. Which option would actually be more useful in your opinion - legend component walls or model walls?

Sorry for the non answer, but both would be best.
I had this conversation with a colleague of mine this very morning! I like the model elements as they are more flexible but requires you to hide them in a 'before everything phase' or a design option so that they don't get in the way but they are much more useful in a template or material reviewing setup. 
The legend component is great for documentation, but only for that.

**Review by dariaivanciucova — APPROVED** [review]

**Pull request merged**